notebooks/en/fine_tuning_llm_for_generate_persian_product_catalogs_in_json_format.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>so many mix of active and passive voice, avoiding passive voice makes it so readable, maybe you could rewrite for clarity? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>You could actually remove this entire paragraph since this notebook doesn't show how to do alignment.</p>